docs/sql-ref-syntax-qry-select-window.md

```diff
@@ -39,17 +39,21 @@ window_function OVER
     * Ranking Functions
 
       **Syntax:** `RANK | DENSE_RANK | PERCENT_RANK | NTILE | ROW_NUMBER`
+      
+      Note: Cannot append `[ { IGNORE | RESPECT } NULLS ]`
 
     * Analytic Functions
 
-      **Syntax:** `CUME_DIST | LAG | LEAD`
+      **Syntax:** `[ CUME_DIST | [ LAG | LEAD | NTH_VALUE | FIRST_VALUE | LAST_VALUE ] [ { IGNORE | RESPECT } NULLS ] ]`
 
     * Aggregate Functions
 
       **Syntax:** `MAX | MIN | COUNT | SUM | AVG | ...`
 
       Please refer to the [Built-in Aggregation Functions](sql-ref-functions-builtin.html#aggregate-functions) document for a complete list of Spark aggregate functions.
 
+      Note: Cannot append `[ { IGNORE | RESPECT } NULLS ]`
+
 * **window_frame**
```

Comment:
Can we add a new section for `{ IGNORE | RESPECT } NULLS`? We need to explain its semantic, the default value, and can apply to which functions.

Reply:
OK